app/controllers/pets_controller.rb

```diff
@@ -11,8 +11,8 @@ def show
   end
 
   def create
-      pet = current_user.pets.create(pet_params)
-      render json: pet
+    pet = current_user.pets.create(pet_params)
+    render json: pet
```

Comment:
I would think some error handling here might be a good idea.

Reply:
Will do